app/services/rapid_ready_for_decision/views/hypertension/about.erb

```diff
@@ -0,0 +1,46 @@
+<h4>About this Document</h4>
+
+<p>
+  The Hypertension Rapid Ready for Decision system retrieves and summarizes
+    VHA medical records related to hypertension claims for increase submitted on va.gov. 
+    VSRs and RVSRs can develop and rate this claim without ordering an exam if there is
+    sufficient existing evidence to show predominance according to 
+    <a href="https://www.ecfr.gov/current/title-38/part-4">DC 7101 (Hypertension) Rating Criteria</a>. 
```

Comment:
How can we make this line more general or do we want to list all CFRs supported by RRD?

Reply:
Working on this in a follow-up PR.